pyproject.toml

```diff
@@ -7,30 +7,28 @@ readme = "README.md"
 packages = [{include = "src"}]
 
 [tool.poetry.dependencies]
-python = ">=3.11,<3.12"
-passpy = "^1.0.2"
-ldap3 = "^2.9.1"
-python-gnupg = "^0.5.1"
+python = ">=3.10,<=3.12.5"
 python-slugify = "^8.0.1"
-pytest-icdiff = "^0.8"
-pytest-deadfixtures = "^2.2.1"
-bagit = "^1.8.1"
 pylint = "^3.1.0"
 mypy = "^1.8.0"
 pytest = "^8.0.2"
 slugify = "^0.0.1"
-typer = "^0.9.0"
-logging = "^0.4.9.6"
 pandas = "^2.2.1"
-openpyxl = "^3.1.2"
 xlrd = "^2.0.1"
 pydantic = "^2.6.3"
 backoff = "^2.2.1"
 requests = "^2.31.0"
-build = "^1.1.1"
 pydantic-settings = "^2.2.1"
-ro-crate-py = {git = "https://github.com/UoA-eResearch/ro-crate-py.git",  branch = "encrypted-metadata"}
 mytardis-rocrate-builder = {git = "https://github.com/JLoveUOA/mytardis_rocrate_builder"}
+ro-crate-py = {git = "https://github.com/UoA-eResearch/ro-crate-py.git",  branch = "encrypted-metadata"}
+openpyxl = "^3.1.5"
+click = "^8.1.7"
+faker = "^27.4.0"
```

Comment:
Do Faker, FactoryBoy and Hypothesis want to sit in general dependencies or should these be dev dependencies? I.e. do we expect our users to run the tests?

Just a thought

Reply:
Moved them to a test group, I don't think I even ended up needing hypothesis and factory boy so will remove those